android/engine/src/main/java/org/smartregister/fhircore/engine/task/WorkflowCarePlanGenerator.kt

```diff
@@ -330,34 +307,23 @@ constructor(
   }
 
   inner class R4PlanDefinitionProcessor
```

Comment:
Any reason to keep creating the Plan Definition Processor like this? FhirOperator should be used instead.

Reply:
This implementation was copied from one of the sample apps to test $apply. We can refactor it to use FHIROperator  instead.